nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -575,6 +583,38 @@ in {
       path = with pkgs; [ file imagemagick ffmpeg ];
     };
 
+
+  systemd.services.mastodon-media-prune =
```

Comment:
Please indent this at the same level as the surrounding context.

Reply:
Also I'd like to have more consistency in regards to the naming here. "Prune" is not mentioned in the upstream docs, right? How about `mastodon-media-auto-remove`?